Garcia-Julz/mine-your-business
# Jg adding 2

# Description
Redirections for login, register, and logout has been altered. Also the register feature is now only asking for your username and pass word.

Next. I have added the ability for a user to add a location to their account so that when a user is trying to create a rig, then can create a location if they don't have any cities saved.

Closes #[Enter issue #]
## Type of change
Please delete options that are not relevant.
# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout jg-adding-2`
4. go ahead and go create a rig in the form. in that form go to the create location and create your location
# Checklist:
